### PR TITLE
I create my content page

### LINKED ISSUE #4: i am creating my content page

### PR DESCRIPTION
**Pull request by Farazbhai**

### CODE REVIEW

File: book-store/Layouts/footer.php [hunk 1]
```diff
@@ -6,15 +6,13 @@
                 <p class="text-light">2005 Stokes Isle Apt. 896,</p>
                 <p class="mt-3 text-light">info@yourdomain.com</p>
                 <p class="mt-3 text-warning">(+68) 120034509</p>
-
```
Comment (MuslimJaan) [inline book-store/Layouts/footer.php:9]: remove empty line

Comment (MuslimJaan) [inline book-store/Layouts/footer.php:9]: remove empty line

File: book-store/Layouts/footer.php [hunk 2]
```diff
@@ -6,15 +6,13 @@
...
             </div>
             <div class="col-sm col-md-3">
                 <span class="mt-5 fw-bold text-info">EXTRA LINKS</span>
                 <p class="mt-3 text-light">Affiliate Program</p>
                 <p class="text-light">Business Services</p>
                 <p class="mt-3 text-light">Education Services</p>
                 <p class="mt-3 text-light">Gift Cards</p>
-
```
Comment (MuslimJaan) [inline book-store/Layouts/footer.php:17]: remove empty line

File: book-store/Layouts/footer.php [hunk 3]
```diff
@@ -23,12 +21,11 @@
                 <p class="mt-3 text-light">Gift Collections</p>
                 <p class="mt-3 text-light">Purchase Cards</p>
                 <span class="mt3 text-light">FAQs</span>
-
```
Comment (MuslimJaan) [inline book-store/Layouts/footer.php:26]: remove empty line

File: book-store/Layouts/footer.php [hunk 4]
```diff
@@ -23,12 +21,11 @@
...
             </div>
             <div class="col-sm col-md-3">
                 <span class="mt-5 fw-bold text-info">BEST SELLERS</span>
                 <div class="d-flex">
-                    <img class="w-25 mt-2" src="Assats/image/9.jpg" alt="">
```
Comment (MuslimJaan) [inline book-store/Layouts/footer.php:31]: image not found in alt

File: book-store/Layouts/footer.php [hunk 5]
```diff
@@ -40,16 +37,13 @@
     <div class="container-fluid bg-black">
         <div class="row">
             <div class="col-sm col-lg-6">
-                <img src="Assats/image/11.jpg" alt="">
-                <img src="Assats/image/10.jpg" alt="">
-                <img src="Assats/image/12.jpg" alt="">
-                <img src="Assats/image/13.jpg" alt="">
+                <img src="assets/images/10.jpg" alt="">
```
Comment (MuslimJaan) [inline book-store/Layouts/footer.php:40]: image not found in alt

File: book-store/Layouts/footer.php [hunk 6]
```diff
@@ -40,16 +37,13 @@
...
+                <img src="assets/images/11.jpg" alt="">
```
Comment (MuslimJaan) [inline book-store/Layouts/footer.php:41]: image not found in alt

File: book-store/Layouts/footer.php [hunk 7]
```diff
@@ -40,16 +37,13 @@
...
+                <img src="assets/images/12.jpg" alt="">
```
Comment (MuslimJaan) [inline book-store/Layouts/footer.php:42]: image not found in alt

File: book-store/Layouts/footer.php [hunk 8]
```diff
@@ -40,16 +37,13 @@
...
+                <img src="assets/images/13.jpg" alt="">
```
Comment (MuslimJaan) [inline book-store/Layouts/footer.php:43]: image not found in alt

File: book-store/home.php [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+      <span class="visually-hidden">Next</span>
+    </button>
+  </div>
+    <div class="container mt-5">
+        <div class="row w-100 text-center">
+        </div>        
+        <div class="container mt-5">
+            <div class="row w-100 text-center">
+                <div class="col-sm col-md-3">
+                    <img class="w-100" src="assets/images/9.jpg" alt="#">
```
Comment (MuslimJaan) [inline book-store/home.php:33]: image not found in alt

File: book-store/home.php [hunk 10]
```diff
@@ -0,0 +1,112 @@
...
+                    <p class="text-center mt-3"><del>$24.00 </del> $22.00</p>
+                    <span class="spn fw-bold">Bulle & Pelle</span>
+                    <p class="pera mt-3">Jimmy Anderson</p>
+                    QTY:<input class="w-25 rounded border-1 " type="number"><br>
+                    <button class="mt-2 rounded border-1 border-black bg-black text-light p-2">Add to
+                        card</button><br>
+                </div>
+                <div class="col-sm col-md-3">
+                    <img class="w-100" src="assets/images/3.jpg" alt="#">
```
Comment (MuslimJaan) [inline book-store/home.php:42]: image not found in alt

File: book-store/home.php [hunk 11]
```diff
@@ -0,0 +1,112 @@
...
+                    <p class="text-center mt-3"><del>$19.00</del> $16.00</p>
+                    <span class="spn fw-bold">Peter and the Wolf</span>
+                    <p class="pera mt-3">ByJOHN WALKER</p>
+                    QTY:<input class="w-25 rounded border-1 " type="number"><br>
+                    <button class="mt-2 rounded border-1 border-black bg-black text-light p-2">Add to
+                        card</button><br>
+                </div>
+                <div class="col-sm col-md-3">
+                    <img class="w-100" src="assets/images/4.jpg" alt="">
```
Comment (MuslimJaan) [inline book-store/home.php:51]: image not found in alt

File: book-store/home.php [hunk 12]
```diff
@@ -0,0 +1,112 @@
...
+                    <p class="text-center mt-3">$24.00</p>
+                    <span class="spn fw-bold">When the Doves</span>
+                    <p class="pera mt-3">ByHOF NURGIN</p>
+                    QTY:<input class="w-25 rounded border-1 " type="number"><br>
+                    <button class="mt-2 rounded border-1 border-black bg-black text-light p-2">Add to
+                        card</button><br>
+                </div>
+                <div class="col-sm col-md-3">
+                    <img class="w-100" src="assets/images/5.jpg" alt="#">
```
Comment (MuslimJaan) [inline book-store/home.php:60]: image not found in alt

File: book-store/home.php [hunk 13]
```diff
@@ -0,0 +1,112 @@
...
+                    <img class="w-100" src="assets/images/6.jpg" alt="#">
+                    <p class="text-center mt-3"><del>$24.00 </del> $22.00</p>
+                    <span class="spn fw-bold">Stuat broud</span>
+                    <p class="pera mt-3">BySAVANNA WALKER</p>
+                    QTY:<input class="w-25 rounded border-1 " type="number"><br>
+                    <button class="mt-2 rounded border-1 border-black bg-black text-light p-2">Add to
+                        card</button><br>
+                </div>
+                <div class="col-sm col-md-3">
+                    <img class="w-100" src="assets/images/7.jpg" alt="#">
```
Comment (MuslimJaan) [inline book-store/home.php:81]: image not found in alt

File: book-store/home.php [hunk 14]
```diff
@@ -0,0 +1,112 @@
...
+                    <p class="text-center mt-3"><del>$19.00</del> $16.00</p>
+                    <span class="spn fw-bold">Peter and the enemy</span>
+                    <p class="pera mt-3">ByJOHN WALKER</p>
+                    QTY:<input class="w-25 rounded border-1 " type="number"><br>
+                    <button class="mt-2 rounded border-1 border-black bg-black text-light p-2">Add to
+                        card</button><br>
+                </div>
+                <div class="col-sm col-md-3">
+                    <img class="w-100" src="assets/images/8.jpg" alt="#">
```
Comment (MuslimJaan) [inline book-store/home.php:90]: image not found in alt

File: book-store/home.php [hunk 15]
```diff
@@ -0,0 +1,112 @@
...
+                    <p class="text-center mt-3">$24.00</p>
+                    <span class="spn fw-bold">When the Doves</span>
+                    <p class="pera mt-3">ByHOF NURGIN</p>
+                    QTY:<input class="w-25 rounded border-1 " type="number"><br>
+                    <button class="mt-2 rounded border-1 border-black bg-black text-light p-2">Add to
+                        card</button><br>
+                </div>
+                <div class="col-sm col-md-3">
+                    <img class="w-100" src="assets/images/1.jpg" alt="#">
```
Comment (MuslimJaan) [inline book-store/home.php:99]: image not found in alt

**Review by Akram-baloch — APPROVED** [review]

**Pull request merged**